### ISSUE TITLE
Hosting: Verify how denied plugins behave when installed via SFTP.

### ISSUE THREAD

**Issue by kwight** [Automattic/wp-calypso#37532]
We have a variety of denied plugins for the Atomic platform, but users will be able to install whatever plugins they like via SFTP. WPcomSH _may_ deactivate them on page load, but this behaviour needs to be verified.

See: p7DVsv-7AO-p2#comment-24434

**Comment by apeatling** [comment]
The current check is not great, but it should do the job as well as it is presently done. I think we can punt this to future.

**Comment by automattic-ian** [comment]
@kwight I agree with @apeatling that this should be punted, because users can install plugins via existing routes that are able to circumvent current measures anyway.
